### ISSUE TITLE
Installed and crashed Piwik Administration

### ISSUE THREAD

**Issue by Cruiser13** [piwikjapan/plugin-clickheat#2]
After installing the plugin I tried to activate it. After that, I was unable to reach my Piwik Administration area (page not found 404 error by the browser).
The default site dashboard was still working but heatmap did not appear.

After removing the clickheat-plugin manually via FTP, the Administration area worked as expected.

CHMod 777 for plugin folder was given.
Plugin was downloaded from the official server, version 0.1.2. Server system is Windows Server 2012 R2 with IIS.

**Comment by piwikjapan** [comment]
Sorry. I made a typo. Do you use ClickHeat http://plugins.piwik.org/ClickHeat not ClickHeart ?

**Comment by piwikjapan** [comment]
And Now, I don't consider Windows. Sorry.

**Comment by Cruiser13** [comment]
Yes, ClickHeat. I can replicate the issue on a *nix system with apache and latest piwik too. Install from Piwik marketplace and get a 404 in Administration of Piwik. No further error message yet.

**Comment by piwikjapan** [comment]
Could you put apache's error log here ?

**Comment by Cruiser13** [comment]
Hello, there is nothing to find in my apache error log, sorry.

**Comment by gfive** [comment]
I have the same problem, also unix and latest piwik. (won't install)

**Comment by piwikjapan** [comment]
Where did you get error 404 on Administrator ?

For example, 

http://yourpiwiksite/index.php?module=CoreAdminHome&action=generalSettings 
(When you clicked link "Administrator", showed this page first.)
http://yourpiwiksite/index.php?module=CorePluginsAdmin&action=plugins
(Manage Plugins - You can install / uninstall plugin on this page.)

or another?

**Comment by gfive** [comment]
When I try to activate the plug-in i get the following error:

The class ClickHeat couldn't be found in the file '/home2/element/public_html/analytics/plugins/ClickHeat/ClickHeat.php'

I have to remove the plug-in so piwik will run again.

**Comment by piwikjapan** [comment]
Muuuuu..... Sorry for I can not find the cause of this serious problem. And 403 error was reported, it is also important, I think I should rewrite start up script on plugin.

**Comment by Cruiser13** [comment]
A solution would be great, thanks for looking into it. Awesome plugin if it works.

**Comment by bisontebiscottato** [comment]
any update on this error? thanks

**Comment by mauricezuhlke** [comment]
Are there any updates on this error? I get the class 'ClickHeat' couldn't be found and then I have to manually delete the plugin folder to use Piwik again.

My setup is:
Apache/2.4.7 (Ubuntu)
PHP 5.5.9-1ubuntu4.11
MySQL 5.5.43-0ubuntu0.14.04.1

And the error message is:

```
The class ClickHeat couldn't be found in the file '/var/www/html/piwik/plugins/ClickHeat/ClickHeat.php'
```

**Comment by piwikjapan** [comment]
Unfortunately still I couldn't find cause of problems.

**Comment by Globulopolis** [comment]
@piwikjapan something went wrong when the plugin published to marketplace. When I install it from marketplace I saw an error as described above. After looking into problem file I see that the content of this file is the same with the config.piwik.php
So I need to copy/paste content of ClickHeat.php from this repo into local file and everything is ok.

Try to retag(add new version tag) for you plugin.

**Comment by sgiehl** [comment]
Maybe it's a problem to have `ClickHeat.php` and `clickheat.php`. It will be a problem on windows for sure (even when using apache instead of IIS)

**Comment by Globulopolis** [comment]
@sgiehl you're right. It's a problem with these to files on windows platform.

**Comment by Cruiser13** [comment]
Would you adjust the filename @piwikjapan ?

**Comment by shenzhuxi** [comment]
I checked out ClickHeat.php from git, then I can get in Piwik Administration. 
But when I went to Visitors > ClickHeat, I got "Fatal error: Cannot redeclare class Piwik\Plugins\ClickHeat\ClickHeat in piwik/piwik/plugins/ClickHeat/clickheat.php on line 0".

**Comment by cssandstuff** [comment]
any update on this? am getting the same error "Fatal error: Cannot redeclare class Piwik\Plugins\ClickHeat\ClickHeat" am using 2.16.1.
@piwikjapan やまちゃんお願いします 🙇
